src/pages/index.js

```diff
@@ -85,43 +127,160 @@ const addModal = new PopUpWithForm(
 editModal.setEventListeners();
 addModal.setEventListeners();
 
+const cardDeletePopUp = new PopUpWithConfirmation({
+  popUpSelector: "#delete-modal",
+});
+
+cardDeletePopUp.setEventListeners();
+
 function handleProfileEditSubmit(inputValues) {
-  userInfo.setUserInfo(inputValues);
-  editModal.close();
+  editModal.setLoading(true);
+  return api
+    .setUserInfo(inputValues.title, inputValues.description)
+    .then(() => {
+      userInfo.setUserInfo(inputValues);
+      profileEditFormValidator.disableButton();
+      editModal.close();
+    })
+    .catch((err) => {
+      console.error(err);
+    })
+    .finally(() => {
+      editModal.setLoading(false);
+    });
 }
 
 function handleImageClick(name, link) {
   previewCardModal.open(name, link);
 }
 
+function handleEditButtonClick() {
+  profileEditModal.classList.add("modal_opened");
+}
+
+if (profileEditButton) {
+  profileEditButton.addEventListener("click", handleEditButtonClick);
```

Comment:
* You haven't deleted something https://github.com/mowooje/se_project_aroundtheus/pull/4#discussion_r1759991746

You can't use `modal_opened` here because you have the `open` method in the modals. And you open the profile modal in line 89 using `editModal.open();`


You have the same click handler in line 89